app/views/clients/status.html.erb

```diff
@@ -1,8 +1,8 @@
 <div class="client-status">  
   <h1>
-    Application Status for: <strong class="text-primary"><%= @client.full_name %></strong><br />
+    <%= I18n.t "client_status.application_status_for" %><strong class="text-primary"><%= @client.full_name %></strong><br />
     <small>
-      Thank you for your interest in LUCHA's services. See the steps below to complete your application.
+      <%= I18n.t "client_status.thank_you" %>
     </small>
```

Comment:
You're right, but that page is for employees and I'm not sure it's in our scope to translate the pages that are not client facing? What do you think is the easiest way to have LUCHA fill in the translations? I was thinking there is probably some tool already built for this very thing.  Here's a few things I found: [i18n-tasks](https://github.com/glebm/i18n-tasks), [Locale](https://www.localeapp.com) - we should look into them or maybe even spin up a quick app we could share?